src/main/java/vectorwing/farmersdelight/common/crafting/CuttingBoardRecipe.java

```diff
@@ -68,10 +69,11 @@ public NonNullList<Ingredient> getIngredients() {
 		return nonnulllist;
 	}
 
+	//This method is unused in regular F'D. For CraftTweaker support, if the tool goes first we can easily wrap/unwrap to and from IIngredient
```

Comment:
This method was previously used in the JEI plugin to establish both ingredients and tool as inputs for recipe lookup, but the new JEI API defines those in a different manner.

In usual circumstances, you would be reminding me to delete an unused method, but if it's useful for CT integration, we can keep it!